README.md

```diff
@@ -47,120 +64,202 @@ Here are the steps for provisioning a new cluster:
         -e "PLATFORM=$PLATFORM" \
         -e "VAULT_PASS=$VAULT_PASS" \
         k8s-provisioner:local /bin/bash provision.sh
-    ```    
-1. `VERY IMPORTANT`: Upload the zip with the TLS assets in the LastPass note from step 2.
-    The zip is found at `credentials/${CLUSTER_NAME}.zip`
-    These initial credentials are vital for subsequent updates in the cluster.
+    ```
 
-The following steps have to be manually done:
+At this point, you have provisioned a Kubernetes cluster without anything running on it. The next steps and involve integrating the cluster and adding services to run on it. Proceed only if you have to create a new complete environment.
```

Comment:
The next step is to.....? I think sounds better? :)

Reply:
Updated.